src/components/ui/form/text-input.tsx

```diff
@@ -0,0 +1,15 @@
+import { DetailedHTMLProps, InputHTMLAttributes } from 'react'
+
+import clsxm from '~/utils/clsxm'
+
+const TextInput = (props: DetailedHTMLProps<InputHTMLAttributes<HTMLInputElement>, HTMLInputElement>) => {
+  const className = clsxm(
+    'mt-2 bg-c-white-purple border border-c-white-purple-2 text-sm rounded-[4px] block w-full p-2.5 outline-none focus:border-c-light-purple text-c-black',
```

Comment:
I think you forgot to change from `c` to `custom`, for example: `bg-c-white-purple`.